bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/MagnitudeBand.scala

```diff
@@ -1,15 +1,19 @@
 package edu.gemini.spModel.core
 
-sealed abstract class MagnitudeBand private (val name: String, val wavelengthMidPointNm: Option[Int], val description: Option[String], val defaultSystem: MagnitudeSystem) extends Product with Serializable {
+sealed abstract class MagnitudeBand private (val name: String, val center: Wavelength, val width: Wavelength, val description: Option[String], val defaultSystem: MagnitudeSystem) extends Product with Serializable {
 
-  private def this(name: String, wavelengthMidPointNm: Int) =
-    this(name, Some(wavelengthMidPointNm), None, MagnitudeSystem.VEGA)
+  private def this(name: String, center: Wavelength, width: Wavelength) =
+    this(name, center, width, None, MagnitudeSystem.VEGA)
 
-  private def this(name: String, wavelengthMidPointNm: Int, description: String) =
-    this(name, Some(wavelengthMidPointNm), Some(description), MagnitudeSystem.VEGA)
+  private def this(name: String, center: Wavelength, width: Wavelength, description: String) =
+    this(name, center, width, Some(description), MagnitudeSystem.VEGA)
 
-  private def this(name: String, wavelengthMidPointNm: Int, description: String, defaultMagnitudeSystem: MagnitudeSystem) =
-    this(name, Some(wavelengthMidPointNm), Some(description), defaultMagnitudeSystem)
+  private def this(name: String, center: Wavelength, width: Wavelength, description: String, defaultMagnitudeSystem: MagnitudeSystem) =
+    this(name, center, width, Some(description), defaultMagnitudeSystem)
+
+  val start: Wavelength = center + width / 2
+
+  val end:   Wavelength = center - width / 2
```

Comment:
I am with @andrewwstephens here: the ```Wavelength``` is just a length which can (currently) be measured in microns or nanometers.

Is this a good moment to take another attempt at promoting http://www.squants.com/ (or some such  thing)? Besides from being able to define stuff like ```1.nm``` and convert between units I also think such a library would be useful for implementing some future tasks where we (I) need to support different units for values in the source definitions.

Regarding how to define the bands: They are defined as midpoint and width in the literature (at least that's true for the Sloan filters which I've looked up), so I am somewhat leaning to use that to define the bands, however if it's preferable to define them by start and end that's ok, too.